README.md

```diff
@@ -22,6 +22,20 @@ Top 5 accuracy: 0.89814
 
 # openvino_quantization
 
+Download ```convert.py``` script:
+https://gist.github.com/dkurt/3486f09fafe568ee20e500310dedcb9f
```

Comment:
Please just insert a content of this script here

Reply:
Can you please check if master branch of OpenVINO will work or not?
```
git clone https://github.com/openvinotoolkit/openvino
python3 ~/openvino/model-optimizer/mo_tf.py ...
```

Reply:
No, it doesn't work